include/z64item.h

```diff
@@ -15,7 +15,7 @@ typedef enum {
     /* 4 */ EQUIP_TYPE_MAX
 } EquipmentType;
 
-// `EquipInv*` enums are for Inventory.equipment (for example used in the `CHECK_OWNED_EQUIP` macro)
+// `EquipInv____` enums are for Inventory.equipment (for example used in the `CHECK_OWNED_EQUIP` macro)
```

Comment:
will say I am not the biggest fan of the `*` notation that has been used alot lately, mainly cause it looks like pointers in the context of a C codebase lol

But also dont think `____` should be used in place of it

Reply:
Well wildcards are a lot more often `*` than not so yeah idk https://en.wikipedia.org/wiki/Wildcard_character

`EquipInvXxx`, `EquipInvXXX`, `EquipInv***`, `EquipInv...`, `EquipInv[...]`, `EquipInv.*`, `EquipInv%` ...